Formula/consul-enterprise.rb

```diff
@@ -34,40 +34,12 @@ def install
     bin.install "consul"
   end
 
-  plist_options manual: "consul agent -dev -bind 127.0.0.1"
-
-  def plist; <<~EOS
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-    <key>KeepAlive</key>
-    <dict>
-        <key>SuccessfulExit</key>
-        <false/>
-    </dict>
-    <key>Label</key>
-    <string>#{plist_name}</string>
-    <key>ProgramArguments</key>
-    <array>
-        <string>#{opt_bin}/consul</string>
-        <string>agent</string>
-        <string>-dev</string>
-        <string>-bind</string>
-        <string>127.0.0.1</string>
-    </array>
-    <key>RunAtLoad</key>
-    <true/>
-    <key>WorkingDirectory</key>
-    <string>#{var}</string>
-    <key>StandardErrorPath</key>
-    <string>#{var}/log/consul.log</string>
-    <key>StandardOutPath</key>
-    <string>#{var}/log/consul.log</string>
-</dict>
-</plist>
-
-EOS
+  service do
+    run [bin/"consul", "agent", "-dev", "-bind", "127.0.0.1"]
+    keep_alive true
```

Comment:
Hey, is this has to be like that `keep_alive successful_exit: false` and also for others to.

Reply:
No one line. I mean like this [Link](https://github.com/buildkite/homebrew-buildkite/pull/31/files)

Reply:
a5a0c454b4ba43b02ad07fc0058e964f84b502a6 should resolve this.